examples/ASCollectionView/Sample/SupplementaryNode.m

```diff
@@ -17,9 +17,11 @@
 
 static CGFloat kInsets = 15.0;
 
-@implementation SupplementaryNode {
-  ASTextNode *_textNode;
-}
+@interface SupplementaryNode ()
```

Comment:
We could use ASTextCellNode for this, but it is pretty basic in the current state. It defines it's own font size, insets etc. without any way to change that values. Maybe worth it to look into add some more ways to configure ASTextCellNode?